skins/greytoken.scss

```
@@ -0,0 +1 @@
+
```

You will need to add you styles in the `.scss` file.
```suggestion
.greytoken {
  --primary-color: #63C194;
  --bg-color: black;
  --text-color: #63C194;
  --link-color: white;
  --heading-color: #BCBEC0;
  --border-color: #63C194;
  --header-bg: black;
}
```